wfe2/wfe.go

```diff
@@ -2337,22 +2337,23 @@ func (wfe *WebFrontEndImpl) NewOrder(
 		names[i] = ident.Value
 	}
 
-	err = policy.WellFormedDomainNames(names)
+	normalizedNames := core.UniqueLowerNames(names)
```

Comment:
Honestly, I'd do:
```suggestion
	names = core.UniqueLowerNames(names)
```

Usually, reusing/shadowing like this would be bad. But the whole point here is that we want to make sure we use the normalized names, and _not_ use the non-normalized names... so having a variable named "names" just sitting there waiting to be used is asking for a mistake in the future. I think overwriting that variable with the normalized names is the safer thing to do.